app/components/OrganizationMemberList.tsx

```diff
@@ -0,0 +1,250 @@
+  const rowHeight = 50
+
+  const getSelectionState = () => {
+    let selectionState: boolean | null = members.at(0)?.isSelected ?? false
+    for (const member of members) {
+      if (member.isSelected !== selectionState) {
+        selectionState = null
+        break
+      }
+    }
+    return selectionState
+  }
+
+  const selectionState = getSelectionState()
+
+  const changeAllSelection = (value: boolean | null) => {
+    if (value === null) {
+      return
+    }
+    members.forEach(member => {
+      member.isSelected = value
+    })
+    // New Array to trigger update
+    setMembers([...members])
+  }
+
+  const changeSingleSelection = (value: boolean, member: OrgMember & { isSelected: boolean }) => {
+    member.isSelected = value
+    setMembers([...members])
+  }
+
+  const removeSelected = () => {
+    setMembers(members.filter(value => !value.isSelected))
+  }
+
+  const singleRemove = (member: OrgMember & {isSelected: boolean}) => {
+    setMembers(members.filter(value => value !== member))
+  }
+
+  const saveChanges = () => {
+    // TODO api call
+  }
+
+  const hasChanges = () => {
+    if (members.length !== organization.members.length) {
+      return true
+    }
+    for (let i = 0; i < members.length; i++) {
+      const mem1 = members[i]
+      const mem2 = organization.members[i]
+      // TODO replace later on with an equality check on the object
+      if (mem1.role !== mem2.role || mem1.name !== mem2.name || mem1.email !== mem2.email || mem1.avatarURL !== mem2.avatarURL) {
+        return true
+      }
+    }
+    return false
+  }
+
+  const disableSaveChanges = !hasChanges()
+
+  const addUser = () => {
+    // TODO remove below for an actual user add
+    const newMember = {
+      name: 'user' + (members.length + 1),
+      role: Role.user,
+      email: `user${(members.length + 1)}@helpwave.de`,
+      isSelected: false,
+      avatarURL: ''
+    }
+    setMembers([...members, newMember])
+    // Automatically go to the last page
+    setCurrentPage(Math.ceil((members.length + 1) / usersPerPage) - 1)
+  }
+
+  const roleClicked = () => {
+    // TODO do something when clicking the role button
+  }
+
+  return (
+    <div className={tw('flex flex-col')}>
+      <div className={tw('flex flex-row justify-between items-center')}>
+        <span className={tw('font-bold font-space')}>{translation.members + ` (${members.length})`}</span>
+        <div className={tw('flex flex-row')}>
+          <button onClick={addUser} className={tw('flex flex-row items-center px-2 mr-2' + buttonStyle)}>
+            <span className={tw('mr-2')}>{translation.addMember}</span><Dropdown/>
+          </button>
+          <button onClick={saveChanges} disabled={disableSaveChanges} className={tx('px-2' + buttonStyle)}>{translation.saveChanges}</button>
+        </div>
+      </div>
+      <table
+        className={tw(`border-spacing-y-[8px] border-separate h-[${8 * (usersPerPage + 2) + rowHeight * usersPerPage + headerHeight}px]`)}>
```

Comment:
You shouldn't need to calculate the height at all, it is really bad practice to do something like this

Also prefer using the tailwind defaults (`border-spacing-y-2`) instead of custom pixel values